src/hooks/useLocalStorage/index.ts

```diff
@@ -0,0 +1,54 @@
+import { useState } from 'react'
+
+interface useLocalStorageHook {
+  setLocalValue: (value: string) => void
+  storedLocalValue: string
+  removeLocalItem: () => void
+}
+export const useLocalStorage = (
+  key: string,
+  initialValue: string
+): useLocalStorageHook => {
+  // State to store our value
+  // Pass initial state function to useState so logic is only executed once
+  const [storedLocalValue, setStoredLocalValue] = useState(() => {
+    if (typeof window === 'undefined') {
+      return initialValue
+    }
+    try {
+      // Get from local storage by key
+      const item = window.localStorage.getItem(key)
+      // Parse stored json or if none return initialValue
+      return item ? `${item}` : initialValue
```

Comment:
Lodash's [defaultTo](https://www.tutorialspoint.com/lodash/lodash_defaultto.htm) looks more cleaner IMO

```suggestion
      return defaultTo(value, initialValue)
```